website/docs/r/servicecatalog_tag_option_resource_association.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Service Catalog"
+layout: "aws"
+page_title: "AWS: aws_servicecatalog_tag_option_resource_association"
+description: |-
+  Manages a Service Catalog Tag Option Resource Association
+---
+
+# Resource: aws_servicecatalog_tag_option_resource_association
+
+Manages a Service Catalog Tag Option Resource Association.
+
+-> A "resource" is either a Service Catalog portfolio or product.
```

Comment:
`~> **Note:**`, `-> **Note:**`, `~> Note:`, ...
We have many ways of calling out notes - I'm not sure which is "best" for the documentation.